crates/uv/src/commands/project/run.rs

```diff
@@ -805,12 +834,18 @@ impl From<&ExternalCommand> for RunCommand {
         let target_path = PathBuf::from(&target);
         if target.eq_ignore_ascii_case("python") {
             Self::Python(args.to_vec())
-        } else if target_path
-            .extension()
-            .is_some_and(|ext| ext.eq_ignore_ascii_case("py"))
-            && target_path.exists()
+        } else if target_path.extension().is_some_and(|ext| {
+            ext.eq_ignore_ascii_case("py") || (cfg!(windows) && ext.eq_ignore_ascii_case("pyw"))
```

Comment:
Should the `||` here be removed? What's the difference between this and the next branch?

Reply:
Oh that's wrong, yes it should be removed.

Reply:
The next branch will use `pythonw.exe` to execute instead of `python.exe`